web/display/cards/index.html

```diff
@@ -184,7 +184,9 @@ <h4 id="time" class="col text-center text-light">12:36</h4>
 										<span class="targetCurrentLp"></span>
 									</div>
 									<div class="col text-right socConfiguredLp">
-										<i class="manualSocSymbol fas fa-edit hide"></i><i class="socSymbol fas fa-car-battery"></i> <span class="socLp">--</span>
+										<i class="manualSocSymbol fas fa-edit hide"></i>
+										<a class="socHasError hide text-red" href="status/status.php"><i class="fas fa-exclamation"></i></a>
```

Comment:
Im Display bitte keinen Link zur Statusseite einfügen. Wenn unbedingt weitere Infos angezeigt werden sollen (gehört meiner Meinung nach nicht in das Display), dann den Bereich der Warnungen entsprechend erweitern.

Reply:
Geändert.